001-IntroToKubernetes/Coach/05-scaling.md

```diff
@@ -13,8 +13,8 @@
 	- Use the Azure portal or CLI to add more nodes to the AKS cluster.
 	- Use the cluster autoscaler to automatically add more nodes to the cluster as resources are needed.
 	- Change the deployment and reduce the needed CPU number from “0.5” to “0.125” (500m to 125m).
-		- This is the preferred solution as long as the application remains responsive!
-		- **NOTE** In the case the last option doesn't work, delete the old pods and reapply the deployments. Kubernetes deploys new pods before tearing down old ones and if we are out of resources, no new pods will be deployed.
+		- In production environment, consider a discussion with the application owner/architect before reducing any resources.
+		- **NOTE** In this Challenge, if the last option doesn't work, delete the old pods and reapply the deployments. Kubernetes deploys new pods before tearing down old ones and if we are out of resources, no new pods will be deployed.
```

Comment:
"delete the old pods" should be "delete the old deployments".  If you delete the pods but not the deployment, K8S will bring new pods up based on the old deployment immediately as it tries to maintain DSC.

Given that this is the coach's guide, it may be worth calling that out as a "learning exercise" for the students.  If they delete pods, and observe them being recreated immediately...  what is going on?  why is this happening?

Reply:
Agreed. ... Chai,  I will make this tiny change since I've got the code handy.